AAT_raspberry/storage.py

```diff
@@ -0,0 +1,20 @@
+filename = 'store'
```

Comment:
Yes I was thinking about that, this was just a very basic first pass at the functionality of the module. When we combine the modules, we may see that it makes more sense to make it a class.